src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -254,14 +254,18 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
                 register={register}
                 name="duration_hours"
               />
-              <label htmlFor="duration_hours">hour(s)</label>
+              <label className="self-center px-1" htmlFor="duration_hours">
```

Comment:
Why are names here with underscores? There is also a nice `generateId()` function that can give a unique id that can be memoised and used instead.